extension/src/popup/components/ModalInfo/index.tsx

```diff
@@ -75,7 +75,7 @@ export const ModalInfo = ({
 
   return (
     <div className={cardClasses}>
-      <Card variant="secondary">
+      <Card>
```

Comment:
in the new design system, we should use the `primary` card, which has just a _little_ different bg color